pkg/resources/procedure.go

```diff
@@ -183,6 +186,10 @@ func Procedure() *schema.Resource {
 		UpdateContext: UpdateContextProcedure,
 		DeleteContext: DeleteContextProcedure,
 
+		CustomizeDiff: customdiff.All(
+			ComputedIfAnyAttributeChanged(FullyQualifiedNameAttributeName, "name"),
```

Comment:
procedure's fully qualified name also depends on the arguments (same for function and external function).

Reply:
Good catch, fixed.